modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -134,8 +168,13 @@ const toggleFullscreen = (editor: Editor, fullscreenState: Cell<any>) => {
     editor.on('remove', cleanup);
 
     fullscreenState.set(newFullScreenInfo);
+    fullscreenRoot.dom.requestFullscreen();
```

Comment:
I believe this won't work on IE or Safari as according to MDN and caniuse these are only exposed via the namespaces variants (eg `webkitRequestFullscreen()` and `msRequestFullscreen()`). I haven't verified, but just noting in case you haven't checked. It also applies to the othe fullscreen APIs.

Reply:
Also worth noting caniuse says full screen can't be activated by keydown or pointerdown. That will likely complicate things.

Reply:
Yep, I'll need to do some testing on IE but you are right that I will need a guard here.

The key command seems to work on Firefox as long as you don't spam it on and off multiple times but I guess Safari could have gone another way there.

Reply:
It may even be worthwhile to create a Sugar module to handle going fullscreen/closing fullscreen to handle the browser differences.

Reply:
Another thing, you can't add a guard here. If it's added to TinyMCE, then it'll need to work on all supported browsers for something like this.

Reply:
Well it adds to the existing fullscreen mode so all the browsers have the original fullscreen (they fill the browser window, just not the monitor) as a fallback when the requestFullscreen() function is not available.

Reply:
This will need to be a setting to enable this, not change the current behaviour. So given it's an entire new feature it should work on all, especially since the other browsers do support it.

Reply:
If you disagree with the above, then please ask in slack, as that's where that'll need to be discussed and it'll need PM input if it's going to work differently on different browsers.

Reply:
I don't disagree. I assumed it would need to be off by default as it is very obviously different behavior.
There is a slack discussion here:
https://jointiny.slack.com/archives/C0HUC9Y4D/p1596418866055300

Reply:
Yeah, I agree it'd need to be off by default. Even when it's enabled though, the editor normally needs to try and behave the same on all supported browsers. If that's not the case, then we'd need to make sure it's okay (hence the PM input) and document the limitation of the setting.

Reply:
I got it working in IE now. In theory it should work in old Edge and old Safari too (though I can't test it myself).

Reply:
The main difference between browsers now is the behaviour on the keyboard shortcut Ctrl+Shift+f as some browsers accept it as a valid trigger for fullscreen (but they rate limit it) and some ignore it completely so only the CSS fullscreen gets triggered.